game/core/src/me/lihq/game/living/controller/PlayerController.java

```diff
@@ -45,6 +45,18 @@ public boolean keyDown(int keycode)
             return true;
         }
 
+        if (keycode == Input.Keys.J)
```

Comment:
Thats fine then :) arguably it should be a separate controller, not in the player controller but its fine at this stage